### PR TITLE
Add latest transactions

### PR DESCRIPTION
**Pull request by matthiasbeyer**
Adds a list of the latest transactions to the dashboard.

By now not configurable, but I guess we won't include config foo in the first release.

### CODE REVIEW

File: app/models/ledger.rb [hunk 1]
```diff
@@ -8,7 +8,7 @@ class Ledger < ActiveRecord::Base
   validates :name, presence: true
 
   def all_transactions
-    transactions + in_transactions
+    (transactions + in_transactions).sort_by { |t| t.created_at }
```
Comment (TheNeikos) [inline app/models/ledger.rb]: Please make either another method of it, or put some SQL down like so:

`Transaction.where('ledger_id = ? OR receiver_id = ?', id, id)`

and then later on you sort it :)

File: app/models/user.rb [hunk 2]
```diff
@@ -11,4 +11,8 @@ def self.login email, password
     User.find_by_email(email).try(:authenticate, password)
   end
 
+  def all_transactions
+    ledgers.map(&:all_transactions).flatten.sort_by { |t| t.created_at }
```
Comment (TheNeikos) [inline app/models/user.rb]: Check this out: http://api.rubyonrails.org/classes/ActiveRecord/Associations/ClassMethods.html#method-i-has_many

Especially the `:source` argument, if you make `all_transactions` a query you should be able to use it as a source :dancer:

Comment (matthiasbeyer) [inline app/models/user.rb]: I don't understand how to use this, sorry. Teach me!

Comment (matthiasbeyer) [inline app/models/user.rb]: After some investigation: Sadly, this does not work.

**Pull request merged**